python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -225,6 +225,46 @@ interface {{ ethernet_interface.name }}
    dot1x authentication failure action traffic drop
 {%             endif %}
 {%         endif %}
+{%         if ethernet_interface.dot1x.aaa.unresponsive is arista.avd.defined %}
+{%             set aaa_config = "dot1x aaa unresponsive" %}
+{%             if ethernet_interface.dot1x.aaa.unresponsive.phone_action is arista.avd.defined or ethernet_interface.dot1x.aaa.unresponsive.action is arista.avd.defined %}
+{%                 set actions = [{'name': 'phone_action', 'config': aaa_config ~ ' phone action'}, {'name': 'action', 'config': aaa_config ~ ' action'}] %}
+{%                 for action in actions %}
+{%                     if ethernet_interface.dot1x.aaa.unresponsive[action.name] is arista.avd.defined %}
+{%                         set aaa_action_config = action.config %}
+{%                         if ethernet_interface.dot1x.aaa.unresponsive[action.name].apply_cached_results is arista.avd.defined(true) or
+                              ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow is arista.avd.defined(true) or
+                              ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_vlan is arista.avd.defined or
+                              ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_access_list is arista.avd.defined %}
+{%                             if ethernet_interface.dot1x.aaa.unresponsive[action.name].apply_cached_results is arista.avd.defined(true) %}
+{%                                 set action_apply_config = "apply cached-results" %}
+{%                                 if ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration is arista.avd.defined and ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration_unit is arista.avd.defined %}
+{%                                     set action_apply_config = action_apply_config ~ " timeout " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration ~ " " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].cached_results_timeout.time_duration_unit %}
+{%                                 endif %}
+{%                             endif %}
+{%                             if ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow is arista.avd.defined(true) %}
+{%                                 set traffic = "traffic allow" %}
+{%                             elif ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_vlan is arista.avd.defined %}
+{%                                 set traffic = "traffic allow vlan " ~ ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_vlan %}
+{%                             elif ethernet_interface.dot1x.aaa.unresponsive[action.name].traffic_allow_access_list is arista.avd.defined %}
```

Comment:
this should not be in elif - we can have both access_list and vlan together 

if we give `traffic_allow_vlan` and `traffic_allow_access_list` both it should concatenate and return a config like -
`dot1x aaa unresponsive action traffic allow vlan 13 access-list ac1`

Reply:
Fixed.